src/Zimbra-1.4.xml

```diff
@@ -0,0 +1,191 @@
+<SPDX name="Zimbra Public License v1.4" identifier="Zimbra-1.4" osi-approved="false">
+  <urls>
+    <url>http://www.zimbra.com/legal/zimbra-public-license-1-4</url>
+  </urls>
+  <license>
+    <title>
+      <p>Zimbra Public License, Version 1.4 (ZPL)</p>
+    </title>
+    <optional>
+      <p>This Zimbra Public License (this “Agreement”) is a legal agreement that describes the terms under which
+         Zimbra, Inc., a Texas corporation (“Zimbra”) will provide software to you via download or otherwise
+         (“Software”). By using the Software, you, an individual or an entity (“You”) agree to the terms of
+         this Agreement.</p>
+      <p>In consideration of the mutual promises and upon the terms and conditions set forth below, the parties
+         agree as follows:</p>
+    </optional>
+    <body>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Grant of Copyright License</p>
+        </li>
+      </list>
+      <list>
+        <li>
+          <b>1.1</b>
+          <p>- Subject to the terms and conditions of this Agreement, Zimbra hereby grants to You, under any
```

Comment:
need to double check whether these dashes are used as bullets, and if so, wrap them in b tags